app/components/common/SearchMap.js

```diff
@@ -0,0 +1,107 @@
+import React, {Component, PropTypes} from 'react';
+import {Gmaps, Marker, InfoWindow, Circle} from 'react-gmaps';
+import Dialog from 'material-ui/lib/dialog';
+import FlatButton from 'material-ui/lib/flat-button';
+
+export default class SearchMap extends Component {
```

Comment:
I'd say you have extra points by implementing the map view. Very few people do that for this test :)